chain/types/ethtypes/eth_transactions.go

```diff
@@ -62,9 +62,14 @@ type EthTxArgs struct {
 // - BlockHash
 // - BlockNumber
 // - TransactionIndex
-// - From
```

Comment:
I'm not sure why we weren't filling this field, but it's easy enough to do so and it saves some work elsewhere.